components/usergroup/meetupForm.tsx

```diff
@@ -10,23 +10,17 @@ type MeetupFormProps = {
 };
 
 export const MeetupForm = (props: MeetupFormProps) => {
-  const { registerUrl, meetupBtnText, meetupTitle } = props;
+  const { registerUrl, meetupBtnText } = props;
```

Comment:
remove `meetupTitle` from props as well since it isn't used anywhere else

Reply:
removed